comp/core/configsync/configsyncimpl/sync.go

```diff
@@ -17,16 +17,17 @@ import (
 	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
 )
 
-func (cs *configSync) updater() {
+func (cs *configSync) updater() error {
+	cs.Log.Debugf("Pulling new configuration from agent-core at '%s'", cs.url.String())
 	cfg, err := fetchConfig(cs.ctx, cs.client, cs.Authtoken.Get(), cs.url.String())
 	if err != nil {
 		if cs.connected {
-			cs.Log.Warnf("Failed to fetch config from core agent: %v", err)
+			cs.Log.Warnf("Loosed connectivity to core-agent to fetch config: %v", err)
```

Comment:
"Loosed connectivity" ? Do you mean Lost?

Reply:
This keeps coming up 😂  This is not my original PR, just backporting. I don't want to touch the code from the original PR/commit, we can fix this in `main` given that it's only a cosmetic/language issue.